dev-php/pecl-uuid/pecl-uuid-1.2.1.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+PHP_EXT_NAME="uuid"
+PHP_EXT_INIT="yes"
+PHP_EXT_ZENDEXT="no"
+DOCS=( CREDITS )
+
+USE_PHP="php8-2"
```

Comment:
Could you test it with 8.3 too?

Reply:
Hey, @thesamesam. 

Sure, I'll test it, too.

BTW, my pkgdev commit omitted a change on profiles/package.mask, to unmask pecl-uuid. Shall I add this change in the same PR?

Cheers,

Reply:
Hi! Yes please, in a second commit for the unmask (same PR) after the one adding 1.2.1.

Reply:
@thesamesam, works fine with php:8.3

$ php php.php 
version: 8.3.14
uuid_create: b29d5988-aca3-4457-8f1b-593ab2ad4a99

Please, check if the order of commits is fine.

Thanks

Reply:
Looks good.